CIP-????/README.md

```diff
@@ -0,0 +1,63 @@
+---
+CIP: ?
+Title: Integration of `keccak256` into Plutus
+Status: Proposed
+Category: Plutus
+Authors: 
+  - Sergei Patrikeev <so.patrikeev@gmail.com>
+  - Iñigo Querejeta-Azurmendi <inigo.querejeta@iohk.io>
+Implementors: []
+Discussions: 
+  - https://github.com/cardano-foundation/CIPs/pull/524
+Created: 2023-06-13
+License: Apache-2.0
+---
+
+## Abstract
+This CIP proposes an extension of the current Plutus functions to provide support for the [`keccak256`](https://keccak.team/files/Keccak-submission-3.pdf) hashing algorithm,
+primarily to ensure compatibility with Ethereum's cryptographic infrastructure.
+
+## Motivation
```

Comment:
```suggestion
## Motivation: why is this CIP necessary?
```
This is a standard title which helps readers & implementors also understand what's being answered here (even if authors understand it) 😎